src/components/JourneySelector.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import '../App.css';
+import { fetchFares } from '../helpers/ApiCallHelper';
+import { DepartureInfo } from '../models/DepartureInfo';
+import DropdownList from './DropdownList';
+
+const JourneySelector: React.FC = () => {
+    const stations = [
+        { crs: 'KGX', name: 'King\'s Cross' }, 
+        { crs: 'SVG', name: 'Stevenage' }, 
+        { crs: 'YRK', name: 'York' }, 
+        { crs: 'DAR', name: 'Darlington' }, 
+        { crs: 'NCL', name: 'Newcastle' },
+    ];
+
+    const [departureValue, setDepartureValue] = useState(stations[0]);
+    const [arrivalValue, setArrivalValue] = useState(stations[0]);
+
+    const journeys: Array<DepartureInfo> = [];
+
+    const searchFares = async () => {
+        fetchFares({ 
+            originStationCrs: departureValue.crs,
+            destinationStationCrs: arrivalValue.crs,
+            outboundDateTime: new Date(), 
+        })
+            .then((value) => {
+                value.json().then(value => (value.outboundJourneys as Array<any>).forEach(outboundJourney => {
```

Comment:
we don't want to be using any types where possible

Reply:
We have changed this block to use await instead of async as its neater, and we stopped using the Array<any> type by creating an interface for the API response JSON.